src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -293,6 +293,19 @@
 GOLANG_STACK_FRAME_FUNCTION_REGEX = re.compile(
     r'^([0-9a-zA-Z\.\-\_\\\/\(\)\*]+)\([x0-9a-f\s,\.]*\)$')
 
+
+# Python specific regular expressions.
+PYTHON_UNHANDLED_EXCEPTION = re.compile(r'^ === Uncaught Python exception: ===$')
```

Comment:
^\s*===

Seems slightly better, will there be always a one character space at start.

Reply:
Likely, yes